api/main.py

```diff
@@ -2,6 +2,7 @@
 from fastapi.responses import RedirectResponse
 from src.routers.health import health_router
 from src.routers.mtcars import data_output
+from src.routers.endpoints import table_router
```

Comment:
for consistency, maybe leave the empty line between blocks
```suggestion
from src.routers.endpoints import table_router

```